### PR TITLE
(fix TypeError): Cannot read property 'timer' of undefined

### PR DESCRIPTION
**Pull request by Mzato0001**
fix (#141 )

### CODE REVIEW

**Comment by 1Computer1** [comment]
Are these two separate changes?

**Comment by Mzato0001** [comment]
wait im wrong something on this commit

**Comment by Papaia** [comment]
Why did you make it only ignore bots when their IDs are in `Client#ownerID`?

**Comment by Mzato0001** [comment]
sorry its just my personal use

**Comment by Mzato0001** [comment]
i forgot that ive created a pull request

File: src/struct/commands/CommandHandler.js [hunk 1]
```diff
@@ -723,7 +723,8 @@ class CommandHandler extends AkairoHandler {
         if (!this.cooldowns.get(id)[command.id]) {
             this.cooldowns.get(id)[command.id] = {
                 timer: this.client.setTimeout(() => {
-                    this.client.clearTimeout(this.cooldowns.get(id)[command.id].timer);
+                    if (this.cooldowns.get(id)[command.id])
```
Comment (1Computer1) [inline src/struct/commands/CommandHandler.js:726]: Add braces please!

**Pull request merged**